File: C/lt_01_estseq/lt01_estseq_26.c
#include <math.h>
#include <stdio.h>
int main() {
  int num1, num2;

  printf("Digite um Numero: ");
  scanf("%d", &num1);

  printf("Digite um Numero: ");
  scanf("%d", &num2);

  if (num1 > num2) {
    float aux;
    aux = num1;
    num1 = num2;
    num2 = aux;
  }

  if (fmodf(num2, num1) == 0) {
    printf("%d e divisivel por %d.\n", num2, num1);
  } else {
    printf("%d nao e divisivel por %d.\n", num2, num1);
  }

  return 0;
}

File: Python/lt_01_estseq/lt01_estseq_26.py
num1=(int(input("Insira o 1º valor: ")))
num2=(int(input("Insira o 2º valor: ")))

if num2>num1:
    aux=num1
    num1=num2
    num2=aux

if num1%num2==0:
    print(num1,"É múltiplo de",num2,".")
else:
    print(num1,"Não é múltiplo de",num2,".")
